src/JLD.jl

```diff
@@ -20,6 +20,11 @@ const name_type_attr = "julia type"
 
 typealias BitsKindOrByteString Union(HDF5BitsKind, ByteString)
 
+if VERSION >= v"0.4.0-dev+5379"
+    # Just rename the uses when we drop 0.3 support
+    const UnionType = Union
+end
+
```

Comment:
NVM. I was going to add all the missing ones to `Compat` but it seems that this one is not that trival. I'll probably not bother about this one then... ;-p